connector/src/main/java/org/geysermc/connector/network/session/GeyserSession.java

```diff
@@ -366,6 +366,12 @@ public void packetReceived(PacketReceivedEvent event) {
                             PacketTranslatorRegistry.JAVA_TRANSLATOR.translate(event.getPacket().getClass(), event.getPacket(), GeyserSession.this);
                         }
                     }
+
+                    @Override
+                    public void packetError(PacketErrorEvent event) {
+                        connector.getLogger().warning("Downstream packet error! " + event.getCause().getMessage());
+                        event.setSuppress(true);
```

Comment:
Add a check here for if debug mode is enabled, and if so, print the stacktrace rather than just the message.

Reply:
I didn't implement that initially because it returned an array and then I'd need to add another import. :P All good if I just `getLogger.debug(stacktrace)` without an if check?

Reply:
Sure.